### PR TITLE
slot-option-feature

### PR DESCRIPTION
**Pull request by talitman**

### CODE REVIEW

**Review by erezinchy — DISMISSED** [review]

File: dist/vue-toastr.esm.js [hunk 1]
```diff
@@ -374,7 +374,7 @@ var script$1 = {
...
+var __vue_render__$1 = function () {var _vm=this;var _h=_vm.$createElement;var _c=_vm._self._c||_h;return _c('div',{class:_vm.classNames,staticStyle:{"display":"block"},style:(_vm.data.style),on:{"click":function($event){return _vm.clicked()},"mouseover":_vm.onMouseOver,"mouseout":_vm.onMouseOut}},[(_vm.progressbar)?_c('toast-progress',{ref:"progressBar",attrs:{"percent":_vm.progressBarPercent}}):_vm._e(),_vm._v(" "),(_vm.data.title)?_c('div',{staticClass:"toast-title",domProps:{"innerHTML":_vm._s(_vm.data.title)}}):_vm._e(),_vm._v(" "),(_vm.data.msg && !_vm.$slots.default)?_c('div',{staticClass:"toast-message",domProps:{"innerHTML":_vm._s(_vm.data.msg)}}):_vm._e(),_vm._v(" "),(_vm.$slots.default)?_c('div',{staticClass:"toast-message"},[_vm._t("default")],2):_vm._e()],1)};
```
Comment (igentifyadmin) [inline dist/vue-toastr.esm.js]: Code Smell Code Smell: Split this 783 characters long line (which is greater than 180 authorized). ([javascript:LineLength](https://sonar.igentify.net/coding_rules#rule_key=javascript%3ALineLength))

[See it in SonarQube](https://sonar.igentify.net/project/issues?id=vuet&pullRequest=2&issues=AWz13RY3AaEKZWHEUhIJ&open=AWz13RY3AaEKZWHEUhIJ)

[ISSUE_KEY]: AWz13RY3AaEKZWHEUhIJ

**Review by igentifyadmin — COMMENTED** [review]
SonarQube analysis found issues:
Bug Bugs: 0
Vulnerability Vulnerabilities: 0
Code Smell Code Smells: 6

[See all issues in SonarQube](https://sonar.igentify.net/project/issues?id=vuet&pullRequest=2&resolved=false)

File: dist/vue-toastr.umd.js [hunk 2]
```diff
@@ -705,13 +705,13 @@
   const __vue_script__$2 = script$2;
 
   /* template */
-  var __vue_render__$2 = function () {var _vm=this;var _h=_vm.$createElement;var _c=_vm._self._c||_h;return _c('div',_vm._l((_vm.list),function(toasts,position){return _c('div',{key:position,class:'toast-container ' + position},_vm._l((toasts),function(toast,index){return _c('toast',{key:index,attrs:{"data":toast}})}),1)}),0)};
+  var __vue_render__$2 = function () {var _vm=this;var _h=_vm.$createElement;var _c=_vm._self._c||_h;return _c('div',_vm._l((_vm.list),function(toasts,position){return _c('div',{key:position,class:'toast-container ' + position},_vm._l((toasts),function(toast,index){return _c('toast',{key:index,attrs:{"data":toast}},[_vm._t("default")],2)}),1)}),0)};
```
Comment (igentifyadmin) [inline dist/vue-toastr.umd.js]: Code Smell Code Smell: Split this 351 characters long line (which is greater than 180 authorized). ([javascript:LineLength](https://sonar.igentify.net/coding_rules#rule_key=javascript%3ALineLength))

[See it in SonarQube](https://sonar.igentify.net/project/issues?id=vuet&pullRequest=2&issues=AWz13RYQAaEKZWHEUhII&open=AWz13RYQAaEKZWHEUhII)

[ISSUE_KEY]: AWz13RYQAaEKZWHEUhII

File: dist/vue-toastr.common.js [hunk 3]
```diff
@@ -701,13 +701,13 @@ var browser = createInjector;
 const __vue_script__$2 = script$2;
 
 /* template */
-var __vue_render__$2 = function () {var _vm=this;var _h=_vm.$createElement;var _c=_vm._self._c||_h;return _c('div',_vm._l((_vm.list),function(toasts,position){return _c('div',{key:position,class:'toast-container ' + position},_vm._l((toasts),function(toast,index){return _c('toast',{key:index,attrs:{"data":toast}})}),1)}),0)};
+var __vue_render__$2 = function () {var _vm=this;var _h=_vm.$createElement;var _c=_vm._self._c||_h;return _c('div',_vm._l((_vm.list),function(toasts,position){return _c('div',{key:position,class:'toast-container ' + position},_vm._l((toasts),function(toast,index){return _c('toast',{key:index,attrs:{"data":toast}},[_vm._t("default")],2)}),1)}),0)};
```
Comment (igentifyadmin) [inline dist/vue-toastr.common.js]: Code Smell Code Smell: Split this 349 characters long line (which is greater than 180 authorized). ([javascript:LineLength](https://sonar.igentify.net/coding_rules#rule_key=javascript%3ALineLength))

[See it in SonarQube](https://sonar.igentify.net/project/issues?id=vuet&pullRequest=2&issues=AWz13RZVAaEKZWHEUhIM&open=AWz13RZVAaEKZWHEUhIM)

[ISSUE_KEY]: AWz13RZVAaEKZWHEUhIM

File: dist/vue-toastr.esm.js [hunk 4]
```diff
@@ -699,13 +699,13 @@ var browser = createInjector;
 const __vue_script__$2 = script$2;
 
 /* template */
-var __vue_render__$2 = function () {var _vm=this;var _h=_vm.$createElement;var _c=_vm._self._c||_h;return _c('div',_vm._l((_vm.list),function(toasts,position){return _c('div',{key:position,class:'toast-container ' + position},_vm._l((toasts),function(toast,index){return _c('toast',{key:index,attrs:{"data":toast}})}),1)}),0)};
+var __vue_render__$2 = function () {var _vm=this;var _h=_vm.$createElement;var _c=_vm._self._c||_h;return _c('div',_vm._l((_vm.list),function(toasts,position){return _c('div',{key:position,class:'toast-container ' + position},_vm._l((toasts),function(toast,index){return _c('toast',{key:index,attrs:{"data":toast}},[_vm._t("default")],2)}),1)}),0)};
```
Comment (igentifyadmin) [inline dist/vue-toastr.esm.js]: Code Smell Code Smell: Split this 349 characters long line (which is greater than 180 authorized). ([javascript:LineLength](https://sonar.igentify.net/coding_rules#rule_key=javascript%3ALineLength))

[See it in SonarQube](https://sonar.igentify.net/project/issues?id=vuet&pullRequest=2&issues=AWz13RY3AaEKZWHEUhIK&open=AWz13RY3AaEKZWHEUhIK)

[ISSUE_KEY]: AWz13RY3AaEKZWHEUhIK

File: dist/vue-toastr.common.js [hunk 5]
```diff
@@ -376,7 +376,7 @@ var script$1 = {
...
+var __vue_render__$1 = function () {var _vm=this;var _h=_vm.$createElement;var _c=_vm._self._c||_h;return _c('div',{class:_vm.classNames,staticStyle:{"display":"block"},style:(_vm.data.style),on:{"click":function($event){return _vm.clicked()},"mouseover":_vm.onMouseOver,"mouseout":_vm.onMouseOut}},[(_vm.progressbar)?_c('toast-progress',{ref:"progressBar",attrs:{"percent":_vm.progressBarPercent}}):_vm._e(),_vm._v(" "),(_vm.data.title)?_c('div',{staticClass:"toast-title",domProps:{"innerHTML":_vm._s(_vm.data.title)}}):_vm._e(),_vm._v(" "),(_vm.data.msg && !_vm.$slots.default)?_c('div',{staticClass:"toast-message",domProps:{"innerHTML":_vm._s(_vm.data.msg)}}):_vm._e(),_vm._v(" "),(_vm.$slots.default)?_c('div',{staticClass:"toast-message"},[_vm._t("default")],2):_vm._e()],1)};
```
Comment (igentifyadmin) [inline dist/vue-toastr.common.js]: Code Smell Code Smell: Split this 783 characters long line (which is greater than 180 authorized). ([javascript:LineLength](https://sonar.igentify.net/coding_rules#rule_key=javascript%3ALineLength))

[See it in SonarQube](https://sonar.igentify.net/project/issues?id=vuet&pullRequest=2&issues=AWz13RZVAaEKZWHEUhIL&open=AWz13RZVAaEKZWHEUhIL)

[ISSUE_KEY]: AWz13RZVAaEKZWHEUhIL

File: dist/vue-toastr.umd.js [hunk 6]
```diff
@@ -380,7 +380,7 @@
...
+  var __vue_render__$1 = function () {var _vm=this;var _h=_vm.$createElement;var _c=_vm._self._c||_h;return _c('div',{class:_vm.classNames,staticStyle:{"display":"block"},style:(_vm.data.style),on:{"click":function($event){return _vm.clicked()},"mouseover":_vm.onMouseOver,"mouseout":_vm.onMouseOut}},[(_vm.progressbar)?_c('toast-progress',{ref:"progressBar",attrs:{"percent":_vm.progressBarPercent}}):_vm._e(),_vm._v(" "),(_vm.data.title)?_c('div',{staticClass:"toast-title",domProps:{"innerHTML":_vm._s(_vm.data.title)}}):_vm._e(),_vm._v(" "),(_vm.data.msg && !_vm.$slots.default)?_c('div',{staticClass:"toast-message",domProps:{"innerHTML":_vm._s(_vm.data.msg)}}):_vm._e(),_vm._v(" "),(_vm.$slots.default)?_c('div',{staticClass:"toast-message"},[_vm._t("default")],2):_vm._e()],1)};
```
Comment (igentifyadmin) [inline dist/vue-toastr.umd.js]: Code Smell Code Smell: Split this 785 characters long line (which is greater than 180 authorized). ([javascript:LineLength](https://sonar.igentify.net/coding_rules#rule_key=javascript%3ALineLength))

[See it in SonarQube](https://sonar.igentify.net/project/issues?id=vuet&pullRequest=2&issues=AWz13RYQAaEKZWHEUhIH&open=AWz13RYQAaEKZWHEUhIH)

[ISSUE_KEY]: AWz13RYQAaEKZWHEUhIH

**Review by erezinchy — APPROVED** [review]
LGTM

**Pull request merged**